### PR TITLE
Added MyBatis, Unitils and Powermock

### PR DESCRIPTION
**Pull request by nicovanbelle**

### CODE REVIEW

**Comment by akullpp** [comment]
Please fix the merge issue.

**Comment by nicovanbelle** [comment]
Fixed the merge conflicts!

File: README.md [hunk 1]
```diff
@@ -395,10 +396,12 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [JMH](http://openjdk.java.net/projects/code-tools/jmh/) - Microbenchmarking tool for the JVM.
 * [JMockit](http://jmockit.org/) - The Mock Anything Toolkit for Java, mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
-* [Mockito](https://github.com/mockito/mockito) - Creation of test double objects in automated unit tests for the purpose of TDD or BDD.
+* [Mockito](http://code.google.com/p/mockito/) - Creation of test double objects in automated unit tests for the purpose of TDD or BDD.
```
Comment (akullpp) [inline README.md]: Wrong URL, please use Github instead.

File: README.md [hunk 2]
```diff
@@ -395,10 +396,12 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
...
+* [Powermock](https://code.google.com/p/powermock/) - Framework that allows you to unit test code normally regarded as untestable.
```
Comment (akullpp) [inline README.md]: Wrong URL, please use Github instead.

**Comment by akullpp** [comment]
Closed due to inactivity, continued in #149.

**Pull request closed without merging**